### ISSUE TITLE
compatibility with @angular 8.1.0-rc.0

### ISSUE THREAD

**Issue by qdouble** [enten/udk#11]
When updating to @next version angular, I'm getting the following error: `An unhandled exception occurred: Cannot read property 'assertCompatibleAngularVersion' of undefined`

The CLI outputs this error log:
```
[error] TypeError: Cannot read property 'assertCompatibleAngularVersion' of undefined
    at buildUniversal (/home/marketstripe/Sites/polysolo-workspace/node_modules/udk/angular/lib/udk-builder.js:25:25)
    at onInput (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular-devkit/architect/src/create-builder.js:145:30)
    at SafeSubscriber.context.inboundBus.subscribe.i [as _next] (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular-devkit/architect/src/create-builder.js:63:29)
    at SafeSubscriber.__tryOrUnsub (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:207:16)
    at SafeSubscriber.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:145:22)
    at Subscriber._next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:89:26)
    at Subscriber.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:66:18)
    at Object.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/observable/onErrorResumeNext.js:22:49)
    at SafeSubscriber.__tryOrUnsub (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:207:16)
    at SafeSubscriber.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:145:22)
    at Subscriber._next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:89:26)
    at Subscriber.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:66:18)
    at ReplaySubject.Subject.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subject.js:60:25)
    at ReplaySubject.nextInfiniteTimeWindow (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/ReplaySubject.js:48:31)
    at Object.next (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/operators/shareReplay.js:33:50)
    at SafeSubscriber.__tryOrUnsub (/home/marketstripe/Sites/polysolo-workspace/node_modules/@angular/cli/node_modules/rxjs/internal/Subscriber.js:207:16)
```

So Version is coming up undefined here:   https://github.com/enten/udk/blob/0f84107c51e377fabb4f67f4e70255fa48b17fd6/angular/lib/udk-builder.ts#L73

**Comment by enten** [comment]
I think the angular devkit have changed.

I will investigate next week and make an update to handle changes.

**Comment by qdouble** [comment]
@enten yeah, it no longer exports a Version class... it's been replaced with this function: `export function assertCompatibleAngularVersion(projectRoot: string, logger: logging.LoggerApi) ` 

https://github.com/angular/angular-cli/blob/0ed8de1e8a8bb39fa58c4a475c2fd56c2aae28fa/packages/angular_devkit/build_angular/src/utils/version.ts

**Comment by enten** [comment]
@qdouble  
Thank you very much for your investigation.

According to your instagation: angular-devkit/build-angular changed as [you described](https://github.com/enten/udk/issues/11#issuecomment-507064079).

I made a [commit to handle the change](https://github.com/enten/udk/commit/fc3222c298465bc927adcf35226aa616fa783cf1) and published [udk v1.1.4](https://www.npmjs.com/package/udk/v/1.1.4).

*Can you try to `npm install --dev udk@1.1.4 && ng build` to check if this issue is resolved?*

**Comment by qdouble** [comment]
@enten just test it and it appears to be working great again. Thanks.
